playbooks/maas-tigkstack-influxdb.yml

```diff
@@ -97,6 +108,10 @@
         - "CREATE RETENTION POLICY {{ influxdb_db_retention_policy }} ON {{ influxdb_db_name }} DURATION {{ influxdb_db_retention }} REPLICATION {{ influxdb_db_replication }}"
         - "CREATE USER {{ influxdb_db_metric_user }} WITH PASSWORD '{{ influxdb_db_metric_password }}'"
         - "GRANT ALL ON {{ influxdb_db_name }} TO {{ influxdb_db_metric_user }}"
+      when:
+        - influxdb_db_name not in influx_dbs.stdout_lines
+      changed_when: true
+
```

Comment:
What is the harm in re-running these steps each time?  While best to not, we do run the (very slight) risk that the db gets provisioned, but the remaining steps are skipped, and can then never be executed until someone manually deletes the db.

Reply:
I would say this PR does not make it idempotent. There can be situations where the DB is created but potentially not the retention policy and we would not repair those scenarios.
I saw no harm in rerunning the statements every time the playbook runs but I also did not test situations where influxdb_db_retention was altered on new run, likely causing an execution error.

Reply:
It was just to appease lint. I'll add the skip lint tag instead.